test/screenshot/infra/lib/selenium-api.js

```diff
@@ -972,6 +977,25 @@ class SeleniumApi {
     return `${this.cli_.colorizeUrl(url)} > ${userAgentColor(userAgent.alias)}`;
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  isBusinessHours_() {
+    const MORNING_UTC_HOURS = 13; // 1pm UTC === 9am EST
+    const EVENING_UTC_HOURS = 1; // 1am UTC === 6pm PST
+    const SATURDAY = 7;
+    const SUNDAY = 0;
+
+    const nowDate = new Date();
+    const nowUtcHours = nowDate.getUTCHours();
+
+    const isWeekDay = nowDate.getDay() > SUNDAY || nowDate.getDay() < SATURDAY;
```

Comment:
This is determining `getDay` by local time. What time zone does Travis run in?

Asking because 12-1am UTC Saturday would currently return false (assuming we fix `SATURDAY` above).

Reply:
Travis runs on EC2 in the US. I'm not sure exactly which region. I'll create a follow up PR to explicitly set the timezone. Thanks!